### DISCUSSION TITLE
Eingabe Tipp beim Werkzeug "Koodinaten/Höhe" ?

### DISCUSSION THREAD

**Discussion by nicole-vlbg** [e-netze/webgis-community#222] [Ideas]
Beim Werkzeug Koordinaten/Höhe ist das Feld beim "Eingabe Tipp" leer.
Nach dem es möglich ist die WGS84 Koordinaten auch im Format x° xx' xx" einzugeben,  was aber für den Anwender auf den ersten Blick nicht erkennbar ist, wäre doch der Tipp dazu sinnvoll.

Können wir den evtl. selbst eingeben, wenn ja wo?

![image](https://github.com/user-attachments/assets/55a94aa8-65ae-4116-b70a-7c08908a5e21)

**Comment by gis-eni** [comment]
Natürlich wäre ein Tipp hier sinnvoll. Dazu muss man in ``etc/coordinates/proj/xy`` eine Datei ``tip.txt`` legen.
In unsrem Installationspacket findest du das eigentlich auch (unter ``Konfiguration`` gibt es ``etc`` Verzeichnis. Darin liegt im ``coordinates`` Verzeichnis eine Ordner ``examples`` mit Beispielen. 
Wenn du noch kein ``etc/coordintes/proj`` Verzeichnis hast, kannst du damit beginnen. Die Datei heißt hier dann zwar ``tip_.txt``, die erkennt WebGIS aber auch (den ``_`` gibts, falls du die Datei selber ändern möchtest und sie bei einem Update nicht überschrieben wird). Wenn du also selber Änderungen vornehmen willst, benenne die Datei dann in ``tip.txt`` um.

**Reply by nicole-vlbg** [reply]
Vielen Dank! Ist schon eingebaut!
